tests/infrastructure/instance_types/test_update_vm_with_instancetype_preference.py

```diff
@@ -88,6 +88,7 @@ def updated_vm_with_instancetype_and_preference(
 
 @pytest.mark.gating
 @pytest.mark.polarion("CNV-9680")
+@pytest.mark.s390x
```

Comment:
I think you can remove it as it tests the validator and arch type has no effect on it.